internal/configuration/validator/const.go

```diff
@@ -169,6 +185,11 @@ var specificErrorKeys = map[string]string{
 	"authentication_backend.file.hashing.parallelism":          "config key incorrect: authentication_backend.file.hashing should be authentication_backend.file.password",
 }
 
+const errFmtSessionSecretRedisProvider = "The session secret is not set but must be set when using the %s session provider"
+const errFmtSessionRedisPortRange = "The port must be between 1 and 65535 the %s session provider"
```

Comment:
```suggestion
const errFmtSessionRedisPortRange = "The port must be between 1 and 65535 for the %s session provider"
```

This reads a little odd without the "for".